### PR TITLE
Local buids opt-in

### PR DESCRIPTION
**Pull request by giohappy**
`docker-compose-local-builds.ym`l can be used an override to build from local Dockerfiles

### CODE REVIEW

**Review by afabiani — APPROVED** [review]

File: docker-compose-local-builds-yml
Comment (ridoo) [inline docker-compose-local-builds-yml:1]: The `yml` extension should be separated from file name with a `.` instead a `-`

Comment (giohappy) [reply docker-compose-local-builds-yml:1]: oh, it was a typo. Thanks @ridoo fixed

**Review by ridoo — COMMENTED** [review]
Instead of adding an additional `docker-compose-local-builds.yml` you could add a `Dockerfile` for each component which just includes `FROM <base_image>` and put a `build: ./docker/<component>` in the normal `docker-compose.yml`.

We did this in our proposal for Docker re-organization. For example:

docker-compose.yml: 
https://github.com/52North/geonode-project/blob/centralize_docker_images/docker-compose.yml#L8-L11

Dockerfile:
https://github.com/52North/geonode-project/blob/centralize_docker_images/docker/geonode/Dockerfile#L1-L2

**Comment by giohappy** [comment]
>Instead of adding an additional docker-compose-local-builds.yml you could add a Dockerfile for each component which just includes FROM <base_image> and put a build: ./docker/<component> in the normal docker-compose.yml.

Sounds good @ridoo. Indeed this is the benefit of using Docker multi-state builds. I'm going to read the other proposal ASAP

**Pull request closed without merging**